tg_bot/modules/stickers.py

```diff
@@ -289,12 +311,20 @@ def kang(update: Update, context: CallbackContext):  # sourcery no-metrics
                 packname,
                 packnum,
                 tgs_sticker=sticker_data if is_animated else None,
-                png_sticker=sticker_data if not is_animated else None,
+                webm_sticker=sticker_data if is_video else None,
+                png_sticker=sticker_data if not is_animated and not is_video else None,
             )
         elif e.message == "Stickers_too_much":
             msg.reply_text("Max packsize reached. Press F to pay respecc.")
         elif e.message == "Invalid sticker emojis":
             msg.reply_text("I can't kang with that emoji!")
+        elif e.message == "Sticker_video_nowebm":

```

Comment:
oo ic there are quite an amount of error types
maybe ill make this a string enum later on